src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/target.py

```diff
@@ -121,7 +129,7 @@ def _target_is_null_path(self):
         self.path = '/deployment' # TODO we will fetch this from parameter store
         responses = self.organizations.dir_to_ou(self.path)
         self._create_response_object(responses)
-
+    #pylint: disable=R0911
```

Comment:
Needs an empty line before the comment too.
```suggestion

    #pylint: disable=R0911
```